Java/java-selenium/README.md

```diff
@@ -0,0 +1,63 @@
+![logo](./docs/logo-java-selenium.png)
+
+# axe DevTools Java Selenium API Example
+
+Using axe DevTools Java Selenium, you can integrate axe DevTools into your existing testing environment. This example project demonstrates how axe DevTools Java Selenium is used to detect accessibility issues and generates reports in CSV, HTML, JSON, and XML formats.
+
+## Prerequisites
+
+Selenium WebDriver
```

Comment:
Is this selenium webdriver as in the binary for chromedriver, geckodriver, etc that needs to be installed separately?